docs/docs/canonicai/chapter9.md

```diff
@@ -0,0 +1,33 @@
+# Stand up a jaseci_ai_kit server locally
+
+In this tutorial, we will discuss how to use uvicorn to stand up a jaseci_ai_kit server locally and quickly
+
+Are you Excited? Let's jump right in.
+
+# Process
+First, you will have to cd into the path of the jaseci_ai_kit module for example let try to stand up cl_summer AI module. From the root jaseci directory run this command in the terminal as follows:
+
+* `cd jaseci_kit/jaseci_kit/modules/cl_summer`
+
+You can CD into any AI models, after you go into the cl_summer directory you will have to stand it up locally and how you do that you will run the following:
+
+* `uvicorn [name_of_file]:serv_actions ` in this case `uvicorn cl_summer:serv_actions`
```

Comment:
So this is one way to run the uvicorn. The other way, which I think we should recommend more, is to run it through the installed jaseci_ai_kit pip package. 

So if you have the jaseci_ai_kit python package installed, you can actually run it like this anywhere without having to be in the jaseci repo
```
uvicorn jaseci_ai_kit.tfm_ner:serv_actions
```

Reply:
Let's include this in here too.